api/package.json

```diff
@@ -24,7 +24,8 @@
     "lint:fix": "eslint . --ext .ts --fix",
     "lint": "eslint . --ext .ts",
     "test:browser": "nyc karma start --single-run",
-    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "test": "ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "test:coverage": "nyc npm run test",
```

Comment:
Personally, I "like" having the test coverage when compiling locally as a guide to what I'm missing....

eg. My preference would be to flip these around
- `test` => full test run (with coverage)
- `test:no-coverage` => just run the tests

But I may just be a bit weird 🤷‍♂️

Reply:
Actually, I may be the weird one having different workflow than most, as I'm running JetBrains WebStorm as an IDE. You're right, it may make sense to flip it around. :slightly_smiling_face: 

pinging @open-telemetry/javascript-approvers, wdyt?

Reply:
What if we run the coverage by default when a single package is tested, but no coverage by default when all packages are tested? I doubt anyone is looking through coverage information when they test all 36 packages.

`/package.json`
```json
{
  "scripts": {
    "test": "lerna run test:nocov",
    "test:cov": "lerna run test"
  }
}
```

`somepackage/package.json`
```json
{
  "scripts": {
    "test": "<test command with coverage>",
    "test:nocov": "<test command without coverage"
  }
}
```